src/daemon/rpmostree-sysroot-upgrader.c

```diff
@@ -1030,6 +1032,31 @@ rpmostree_sysroot_upgrader_prep_layering (RpmOstreeSysrootUpgrader *self,
 }
 
 /**
+ * rpmostree_sysroot_upgrader_deploy_set_kargs:
+ * @self: Self
+ * @kernel_args: A list of strings representing kernel_arguments
+ * @cancellable: Cancellable
+ * @error: Error
+ *
+ * A wrapper function that collects the kernel arguments then call the rpmostree_sysroot_upgrader_deploy
+ * to write a pending deployment to the disk
+ *
+ * Returns: FALSE if rpmostree_sysroot_upgrader_deploy fails
+ *
+ */
+gboolean
+rpmostree_sysroot_upgrader_deploy_set_kargs (RpmOstreeSysrootUpgrader *self,
```

Comment:
I'm OK with this, but I think down the line we should add the kernel args to `rpmostree_sysroot_upgrader_deploy()` and have the other callers pass `NULL`.

Reply:
hey thanks for the review :). For this part, I believe other callers are still able to set NULL for the kernel args, as the kargs_strv is implemented as one of the attributes from the upgrader, similar to other attributes as well. 
https://github.com/projectatomic/rpm-ostree/pull/1013/files#diff-af8b24e3bf31175f007dd70d4b1a46fcR83

My personal opinion, but I could be wrong, is .. it feels a bit unnecessary to change other code when you can just set NULL to the kargs_strv attribute prior to calling `rpmostree_sysroot_upgrader_deploy`? ( I kinda tried to avoid changing too much existing code base too =) ). But I am also fine if you want to change the function.